src/Extensions/DBHTMLFieldExtension.php

```diff
@@ -4,18 +4,28 @@
 namespace SilverStripe\SearchService\Extensions;
 
 use SilverStripe\Core\Extension;
+use SilverStripe\SearchService\Service\IndexConfiguration;
+use SilverStripe\SearchService\Service\Traits\ConfigurationAware;
 
 class DBHTMLFieldExtension extends Extension
 {
+    use ConfigurationAware;
```

Comment:
If you want to do it with injection, I think you'll need another trait here. Extensions aren't injectable by default.

```suggestion
    use ConfigurationAware;
    use Injectable;
```

Reply:
Oh I saw this after I pushed a different approach - is this more idiomatic?